openwisp_controller/config/static/config/js/tabs.js

```diff
@@ -23,6 +23,7 @@ django.jQuery(function ($) {
         type: 'tabshown',
         tabId: tabId,
       });
+      window.dispatchEvent(new Event('tabchange'));
```

Comment:
why do you do this? An event is already being triggered with jQuery, just use jQuery